git/util.py

```diff
@@ -935,7 +950,7 @@ def _obtain_lock_or_raise(self) -> None:
             )
 
         try:
-            with open(lock_file, mode='w'):
+            with open(lock_file, mode="w"):
```

Comment:
I have seen various contributors fight over these and do wonder who will win ;).

Reply:
`black` prefers `"`, and some developers automatically runs black in their editor :)

Reply:
Yeah, I ran `black` on the file I was changing, which is how that change came in.

Reply:
I see, these are tool-based preferences by now. Is it possible to drop a `black.config` file in the root to configure this? I do prefer to keep it the way it is, personally.

Reply:
No, `black` is opinionated code formatter. One of the reasons why we like it. We all have to sacrifice something in the name of unity, so we leave the decision to the formatting overlord. :)